Packs/iDefense/Integrations/FeediDefense/FeediDefense.yml

```
@@ -0,0 +1,172 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: iDefense Feed
+  version: -1
+configuration:
+- defaultvalue: 'true'
+  display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- display: API Key
+  hidden: false
+  name: api_token
+  required: true
+  type: 4
+- additionalinfo: Indicators from this integration instance will be marked with this
+    reputation
+  defaultvalue: feedInstanceReputationNotSet
```

it should be bad AFAIK
```suggestion
  defaultvalue: Bad
```